src/main/java/com/google/devtools/build/lib/vfs/ReadonlyFileSystem.java

```diff
@@ -65,11 +65,6 @@ public boolean supportsHardLinksNatively(PathFragment path) {
     return false;
   }
 
-  @Override
```

Comment:
This is removed accidentally? It's causing some test failures internally, but I can add this back. 
Should be 
```
  @Override
  public boolean mayBeCaseOrNormalizationInsensitive() {
    return false;
  }
```
right?

Reply:
Ah, yes, please do!